[OSDOCS-4013] added suggested sentence to change management policy info (#52186)
openshift/openshift-docs: modules/policy-change-management.adoc

* Scaling the cluster persistent storage
* Upgrading the cluster

You can implement a maintenance exclusion by avoiding changes in {cluster-manager} for the following components:

* Deleting a cluster

> * Adding, modifying or removing identity providers

Reviewer: Change to: Adding, modifying, or removing identity providers
Per  the ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series

> * Adding, modifying or removing a user from an elevated group

Reviewer: Change to: Adding, modifying, or removing a user from an elevated group
Per the ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series

* Installing or removing add-ons
* Modifying cluster networking configurations
* Adding, modifying, or removing machine pools
* Enabling or disabling user workload monitoring
* Initiating an upgrade

> 

Reviewer 2: How about cluster history and support?

Reviewer 3: Cluster history is read-only by customers, and has no impact of cluster actions if something changes. Same with support, there are no impact to clusters by modifying anything on the support tab.

Cluster history and support aren't tabs/actions that need to be considered for a maintenance exclusion.

Some changes are released to production incrementally, using feature flags to control availability of new features to specified clusters or customers.

You can implement a maintenance exclusion by avoiding changes in {cluster-manager} for the following components:

> * Deleting a cluster
> * Adding or removing an identity provider
> * Adding or removing a user from an elevated group
> * Changing cluster networking visibility
> * Adding or removing machine pools
> * Initiating an upgrade

Reviewer 3: ```suggestion
* Deleting a cluster
* Adding, modifying or removing an identity provider
* Adding, modifying or removing a user from an elevated group
* Installing or removing add-ons
* Modifying cluster networking configurations
* Adding, modifying, or removing machine pools
* Enabling or disabling user workload monitoring
* Initiating an upgrade
```

> * Defining upgrade policies

Reviewer 4: This should be "Initiating an upgrade"... Defining an upgrade policy, such as changing from Automatic to Manual, is actually implied as part of the maintenance exclusion process in the call-out below.

Author: Thanks, Bill. I've updated the docs with the suggestion.

[IMPORTANT]
====

> To enforce the maintenance exclusion, ensure autoscaling or automatic upgrade policies have been disabled. After the maintenance has completed, proceed with enabling autoscaling or automatic upgrade policies as desired.

Reviewer 3: ```suggestion
To enforce the maintenance exclusion, ensure machine pool autoscaling or automatic upgrade policies have been disabled. After the maintenance exclusion has been lifted, proceed with enabling machine pool autoscaling or automatic upgrade policies as desired.
```

Some changes are released to production incrementally, using feature flags to control availability of new features to specified clusters or customers.

You can implement a maintenance window by avoiding changes in {cluster-manager} for the following components:

> * Creating a cluster

Reviewer 3: ```suggestion
```

Author: Updated in new commit

* Deleting a cluster
* Adding or removing an identity provider
* Adding or removing a user from an elevated group

> * Configuring cluster privacy

Reviewer 3: ```suggestion
* Changing cluster networking visibility
```

Author: Updated in new commit

> * Adding or removing machine pools and configure autoscaling

Reviewer 3: ```suggestion
* Adding or removing machine pools
* Enabling autoscaling on machine pools
```

Author: Updated in new commit

* Defining upgrade policies

[IMPORTANT]
====

> To implement a maintenance window, make sure to turn off auto scaling. After the maintenance has completed, turn auto scaling back on.

Reviewer 3: ```suggestion
To enforce a maintenance window, ensure autoscaling or automatic upgrade policies have been disabled. After the maintenance has completed, proceed with enabling autoscaling or automatic upgrade policies as desired.
```

Author: Updated in new commit with @billmvt suggestion from below as well.
